pnpm nx install failing to create node_modules or angular project

When trying to create a workspace using the pnpm package manager the install fails to create a node_modules/ folder or setup the initial Anuglar project. 

## Current Behavior

Most of the workspace is created though, including .vscode, apps, libs, tools and angular.json. I also get the error message during the 
install.

 >  NX   Nx CLI is not installed globally.


Another oddity is that the *pnpm-lock.yaml* is there and it would look like all the installations are there, but the project folder doesn't even show up until nx starts to setup the workspace. Angular.json also has references to the todo project.


## Expected Behavior

That this would work as it does when the package manager is not specified.

## Steps to Reproduce

I've tried the following commands all with the same failure.
```
npx create-nx-workspace@latest --pm=pnpm --name=nxtest --preset=angular --appName=todo --style=scss --nxCloud=false
pnpm dlx create-nx-workspace@latest --pm=pnpm --name=nxtest --preset=angular --appName=todo --style=scss --nxCloud=false
pnpm create nx-workspace@latest -- --pm=pnpm --name=nxtest --preset=angular --appName=todo --style=scss --nxCloud=false
```

I am actually trying to follow this example: https://nx.dev/angular-tutorial/01-create-application, but with pnpm.


### Failure Logs

I have also tried to work around the initial install

```
cd nxtest
pnpm intsall
```
So far so good. At this point I try to create the TODO Angular app:
```
nx g @nrwl/angular:application todo
```
This works but throws up this throws up a bunch of install warnings about missing peers:
```
 WARN  `node_modules` is present. Lockfile only installation will make it out-of-date
 WARN  deprecated source-map-resolve@0.6.0: See https://github.com/lydell/source-map-resolve#deprecated
 ERR_PNPM_PEER_DEP_ISSUES  Unmet peer dependencies

.
├─┬ @nrwl/cypress
│ ├─┬ @cypress/webpack-preprocessor
│ │ ├── ✕ missing peer @babel/core@^7.0.1
│ │ ├── ✕ missing peer @babel/preset-env@^7.0.0
│ │ ├── ✕ missing peer babel-loader@^8.0.2
│ │ └── ✕ missing peer webpack@"^4 || ^5"
│ ├─┬ ts-loader
│ │ └── ✕ missing peer webpack@^5.0.0
│ └─┬ fork-ts-checker-webpack-plugin
│   └── ✕ missing peer webpack@">= 4"
├─┬ jest-preset-angular
│ └─┬ ts-jest
│   └── ✕ missing peer @babel/core@">=7.0.0-beta.0 <8"
├─┬ ts-jest
│ └── ✕ missing peer @babel/core@">=7.0.0-beta.0 <8"
└─┬ @nrwl/angular
  ├─┬ @nrwl/cypress
  │ └─┬ @cypress/webpack-preprocessor
  │   ├── ✕ missing peer @babel/core@^7.0.1
  │   ├── ✕ missing peer @babel/preset-env@^7.0.0
  │   └── ✕ missing peer babel-loader@^8.0.2
  ├─┬ jasmine-marbles
  │ └── ✕ unmet peer rxjs@^6.5.3: found 7.4.0
  └─┬ rxjs-for-await
    └── ✕ unmet peer rxjs@^6.0.0: found 7.4.0
Peer dependencies that should be installed:
  @babel/core@">=7.0.1 <8.0.0"        babel-loader@">=8.0.2 <9.0.0"       
  @babel/preset-env@">=7.0.0 <8.0.0"  webpack@">=5.0.0 <6.0.0"            

Progress: resolved 1217, reused 1167, downloaded 1, added 0
Command failed: pnpm install --no-frozen-lockfile
```
At this point trying to serve Todos fails:
```
nx serve todos
 >  NX   Cannot find project 'todos'
```
Installing the missing peer dependencies also does not appear to resolve the issue:
```
pnpm add -D @babel/core @babel/preset-env babel-loader webpack
```

### Environment

OS: Manjaro Gnome with latest updates as of 22 May 2022
Shell: Tried both ZSH and BASH
Alias: *alias pnx="pnpm nx"* setup for ZSH and *alias pnx="pnpm nx --" for BASH
Global PNPM installs: 
```
Legend: production dependency, optional only, dev only

/home/jking/.local/share/pnpm/global/5

dependencies:
@electron-forge/cli 6.0.0-beta.63
@nrwl/cli 14.1.7
typescript 4.6.4
```

I am going to continue without PNPM for now,

--- a/packages/cypress/package.json
+++ b/packages/cypress/package.json
@@ -34,6 +34,10 @@
   },
   "dependencies": {
     "@cypress/webpack-preprocessor": "^5.9.1",
+    "@babel/core": "^7.0.1",
+    "@babel/preset-env": "^7.0.0",
+    "babel-loader": "^8.0.2",
+    "webpack": "^4 || ^5",
     "@nrwl/devkit": "file:../devkit",
     "@nrwl/linter": "file:../linter",
     "@nrwl/workspace": "file:../workspace",

--- a/scripts/depcheck/discrepancies.ts
+++ b/scripts/depcheck/discrepancies.ts
@@ -5,6 +5,7 @@ import { satisfies } from 'semver';
 const IGNORE_MATCHES = {
   '*': [],
   angular: ['webpack-merge', '@phenomnomnominal/tsquery'],
+  cypress: ['webpack'],
 };
 
 export default function getDiscrepancies(

